hello-world/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>2.33</version>
+        <relativePath />
+    </parent>
+    <artifactId>${artifactId}</artifactId>
+    <version>${version}</version>
+    <packaging>hpi</packaging>
+    <properties>
+        <jenkins.version>2.7.3</jenkins.version>
+        <workflow-step-api-plugin.version>2.12</workflow-step-api-plugin.version>
+        <workflow-support-plugin.version>2.14</workflow-support-plugin.version>
+        <workflow-cps-plugin.version>2.39</workflow-cps-plugin.version>
+    </properties>
+    <name>TODO Plugin</name>
+    <description>TODO</description>
+    <url>https://wiki.jenkins-ci.org/display/JENKINS/TODO+Plugin</url>
+    <licenses>
```

Comment:
As in https://github.com/jenkinsci/maven-hpi-plugin/pull/74 you should offer a comment here I suppose.

Reply:
Similarly commented-out sections for `developers`, `scm`, and explanation of `jenkins.version`.

Reply:
done